charts/eraser/values.yaml

```diff
@@ -69,7 +69,7 @@ runtimeConfig:
         # timeout:
         #   total: 23h
         #   perImage: 1h
-    eraser:
```

Comment:
this shouldn't need to be updated. it will be updated automatically when we do a release